cmd/get_logs_test.go

```diff
@@ -133,6 +133,19 @@ func TestGetLogsCmdValidateArgs(t *testing.T) {
 			expectedErr: errors.Errorf("specified --api-model does not exist (%s)", missingFile),
 			name:        "NeedsAPIModel",
 		},
+		{
+			glc: &getLogsCmd{
+				apiModelPath:           existingFile,
+				linuxSSHPrivateKeyPath: existingFile,
+				linuxScriptPath:        existingFile,
+				windowsScriptPath:      existingFile,
+				sshHostURI:             "server.example.com",
+				location:               "southcentralus",
+				storageContainerSASURL: "https://blob-service-uri/?sas-token",
+			},
+			expectedErr: errors.Errorf("error validating storage container SAS URL https://blob-service-uri/?sas-token. Expected SAS URL format is https://{blob-service-uri}/{container-name}?{sas-token}"),
+			name:        "InvalidSASURL",
```

Comment:
when writing unit tests, you typically include both positive and negative cases

Reply:
updated